### PR TITLE
RS/YJ/Rule 19-3

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**

### CODE REVIEW

File: rct229/rulesets/ashrae9012019/section19/section19rule3.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    def manual_check_required(self, context, calc_vals=None, data=None):
+        rpd_b = context.BASELINE_0
+        weather_b = rpd_b["weather"]
+
+        return not (
+            weather_b.get("cooling_design_day_type") != COOLING_DESIGN_DAY.COOLING_1_0
+            or weather_b.get("heating_design_day_type")
+            != HEATING_DESIGN_DAY.HEATING_99_6
+        )
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/section19/section19rule3.py:36]: You do not need this `manual_check_required` in the partial rule definition class.
Also, the `get_manual_check_required_msg` shall be after the `applicability_check` function. See the order in the function [evaluate](https://github.com/pnnl/ruleset-checking-tool/blob/develop/rct229/rule_engine/rule_base.py#L175-L185)

**Review by weilixu — APPROVED** [review]
approved with one comment need address.

File: rct229/ruletest_engine/run_ruletests.py [hunk 2]
```diff
@@ -194,7 +194,7 @@ def run_test_one_jsontest(test_json):
 # run_airside_tests()
 # run_hvac_general_tests()
 
-# run_test_one_jsontest("ashrae9012019/section22/rule_22_12.json")
+run_test_one_jsontest("ashrae9012019/section19/rule_19_3.json")
```
Comment (weilixu) [inline rct229/ruletest_engine/run_ruletests.py:197]: Comment out this test.

File: rct229/rulesets/ashrae9012019/section19/section19rule3.py (hunk 1 above)
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section19/section19rule3.py:36]: Good point! Addressed.

File: rct229/ruletest_engine/run_ruletests.py (hunk 2 above)
Comment (yunjoonjung-PNNL) [reply rct229/ruletest_engine/run_ruletests.py]: Addressed.

**Review by weilixu — APPROVED** [review]

**Pull request merged**